package.json

```diff
@@ -103,6 +104,7 @@
     "d3-scale": "^4.0.2",
     "d3-selection": "^3.0.0",
     "d3-shape": "^3.1.0",
-    "d3-zoom": "^3.0.0"
+    "d3-zoom": "^3.0.0",
+    "ramda": "^0.28.0"
```

Comment:
Unsure if it's worth adding `ramda` when only `filter` & `map` is used from that package. 🤔